src/python/tests/core/crash_analysis/stack_parsing/stack_analyzer_test.py

```diff
@@ -1321,6 +1321,17 @@ def test_asan_double_free(self):
                                   expected_state, expected_stacktrace,
                                   expected_security_flag)
 
+  def test_libfuzzer_deadly_signal(self):
+    data = self._read_test_data('libfuzzer_deadly_signal.txt')
+    expected_type = 'deadly signal'
+    expected_state = 'NULL'
+    expected_address = ''
+    expected_stacktrace = data
+    expected_security_flag = True
```

Comment:
Is this really security implication, probably not, change crash_analyzer to skip this crash type.

Reply:
Good point, +1 to not security by default.

Reply:
Fyi, you need to add this in https://github.com/google/clusterfuzz/blob/master/src/python/crash_analysis/crash_analyzer.py#L233 to return False, then this test should be "expected_security_flag = False"